internal/cmd/gocdk/main.go

```diff
@@ -95,11 +98,22 @@ func osProcessContext() (*processContext, error) {
 		workdir: workdir,
 		env:     os.Environ(),
 		stdin:   os.Stdin,
-		stdout:  os.Stdout,
-		stderr:  os.Stderr,
+		outlog:  log.New(os.Stdout, "", 0),
+		errlog:  log.New(os.Stderr, "gocdk: ", 0),
 	}, nil
 }
 
+// newTestProcessContext returns a processContext for use in tests.
+func newTestProcessContext(workdir string) *processContext {
+	return &processContext{
+		workdir: workdir,
+		env:     os.Environ(),
+		stdin:   strings.NewReader(""),
+		outlog:  log.New(ioutil.Discard, "", 0),
+		errlog:  log.New(ioutil.Discard, "gocdk: ", 0),
```

Comment:
nit: AFAICT, no prefix required for a log that will be discarded.

Reply:
Done.